op-deployer/pkg/deployer/standard/standard.go

```diff
@@ -197,6 +197,17 @@ func GuardianAddressFor(chainID uint64) (common.Address, error) {
 	}
 }
 
+func ProtocolVersionsOwnerFor(chainID uint64) (common.Address, error) {
+	switch chainID {
+	case 1:
+		return common.HexToAddress("0x847B5c174615B1B7fDF770882256e2D3E95b9D92"), nil
```

Comment:
Protocol Versions address is [0x8062AbC286f5e7D9428a0Ccb9AbD71e50d93b935](https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/configs/mainnet/superchain.toml#L2). 

```
cast call 0x8062AbC286f5e7D9428a0Ccb9AbD71e50d93b935 'owner()(address)' -r $ETH_RPC_URL
0x847B5c174615B1B7fDF770882256e2D3E95b9D92
```